backend/routes/passportLogin.js

```diff
@@ -27,14 +27,20 @@ passport.use(new Strategy({
   callbackURL: 'http://localhost:8080/auth/github/callback'
 }, function(accesstoken, refreshToken, profile, done) {
     User.findOne({ gihubId: profile.id }, function(err, user) {
-      console.log(profile);
       
       if (err) return done(err);
       
       if (user) {
+        console.log('done')
         return done(err, user);
       } else {
-        user = new User({ githubId: profile.id })
+        user = new User({ 
```

Comment:
Do some users have email publicly available? Do we want to try and include email in the User schema?